Failing to run tests due to f-strings

Tests failing due to bad f-strings.

<details>

<summary>Test report</summary>

```
____________________________________________________________________________________________ ERROR collecting tests/gui/test_plot_window.py ____________________________________________________________________________________________
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/python.py:498: in importtestmodule
    mod = import_path(
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/pathlib.py:549: in import_path
    mod = _import_module_using_spec(
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/pathlib.py:725: in _import_module_using_spec
    spec.loader.exec_module(mod)  # type: ignore[union-attr]
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:186: in exec_module
    exec(co, module.__dict__)
tests/gui/test_plot_window.py:12: in <module>
    from EVA.gui.windows.elemental_analysis.elemental_analysis_window import ElementalAnalysisWindow
src/EVA/gui/windows/elemental_analysis/elemental_analysis_window.py:6: in <module>
    from EVA.gui.windows.elemental_analysis.elemental_analysis_presenter import ElementalAnalysisPresenter
src/EVA/gui/windows/elemental_analysis/elemental_analysis_presenter.py:11: in <module>
    from EVA.gui.windows.elemental_analysis.elemental_analysis_view import ElementalAnalysisView
E     File "/home/jacob/EVA/src/EVA/gui/windows/elemental_analysis/elemental_analysis_view.py", line 115
E       e_item.setText(4, f"{energy["diff"]:.3f} keV")
E                                    ^^^^
E   SyntaxError: f-string: unmatched '['
_________________________________________________________________________________________ ERROR collecting tests/system/test_model_spectra.py __________________________________________________________________________________________
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/python.py:498: in importtestmodule
    mod = import_path(
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/pathlib.py:549: in import_path
    mod = _import_module_using_spec(
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/pathlib.py:725: in _import_module_using_spec
    spec.loader.exec_module(mod)  # type: ignore[union-attr]
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:177: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:357: in _rewrite_test
    tree = ast.parse(source, filename=strfn)
../.pyenv/versions/3.10.14/lib/python3.10/ast.py:50: in parse
    return compile(source, filename, mode, flags,
E     File "/home/jacob/EVA/tests/system/test_model_spectra.py", line 41
E       f"Missing transition for {transition["element"]}: {modelled_transition}"
E                                             ^^^^^^^
E   SyntaxError: f-string: unmatched '['
________________________________________________________________________________________ ERROR collecting tests/system/test_run_corrections.py _________________________________________________________________________________________
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/python.py:498: in importtestmodule
    mod = import_path(
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/pathlib.py:549: in import_path
    mod = _import_module_using_spec(
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/pathlib.py:725: in _import_module_using_spec
    spec.loader.exec_module(mod)  # type: ignore[union-attr]
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:177: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:357: in _rewrite_test
    tree = ast.parse(source, filename=strfn)
../.pyenv/versions/3.10.14/lib/python3.10/ast.py:50: in parse
    return compile(source, filename, mode, flags,
E     File "/home/jacob/EVA/tests/system/test_run_corrections.py", line 53
E       f"{" and ".join(e_corr_which)}")
E            ^^^
E   SyntaxError: f-string: expecting '}'
______________________________________________________________________________________________ ERROR collecting tests/system/test_srim.py ______________________________________________________________________________________________
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/python.py:498: in importtestmodule
    mod = import_path(
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/pathlib.py:549: in import_path
    mod = _import_module_using_spec(
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/pathlib.py:725: in _import_module_using_spec
    spec.loader.exec_module(mod)  # type: ignore[union-attr]
../.pyenv/versions/3.10.14/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:186: in exec_module
    exec(co, module.__dict__)
tests/system/test_srim.py:3: in <module>
    from EVA.gui.windows.srim.trim_model import TrimModel
E     File "/home/jacob/EVA/src/EVA/gui/windows/srim/trim_model.py", line 561
E       return os.path.join(f"{get_config()["general"]["working_directory"]}", f"SRIM_{momentum}_MeVc.zip")
E                                            ^^^^^^^
E   SyntaxError: f-string: unmatched '['
======================================================================================================= short test summary info ========================================================================================================
ERROR tests/gui/test_plot_window.py
ERROR tests/system/test_model_spectra.py
ERROR tests/system/test_run_corrections.py
ERROR tests/system/test_srim.py
```
</details>

--- a/src/EVA/gui/windows/elemental_analysis/elemental_analysis_view.py
+++ b/src/EVA/gui/windows/elemental_analysis/elemental_analysis_view.py
@@ -112,7 +112,7 @@ class ElementalAnalysisView(BaseView, Ui_elemental_analysis):
                     e_item = QTreeWidgetItem()
                     e_item.setText(2, energy["element"])
                     e_item.setText(3, energy["transition"])
-                    e_item.setText(4, f"{energy["diff"]:.3f} keV")
+                    e_item.setText(4, f"{energy['diff']:.3f} keV")
                     p_item.addChild(e_item)
 
             items.append(item)

--- a/src/EVA/gui/windows/srim/trim_model.py
+++ b/src/EVA/gui/windows/srim/trim_model.py
@@ -558,7 +558,7 @@ class TrimModel(QObject):
             momentum = "all"
         else:
             momentum = f"{momentum:.5f}"
-        return os.path.join(f"{get_config()["general"]["working_directory"]}", f"SRIM_{momentum}_MeVc.zip")
+        return os.path.join(f"{get_config()['general']['working_directory']}", f"SRIM_{momentum}_MeVc.zip")
 
     def save_sim(self, path: str, rows: list | int | None = None):
         if isinstance(rows, int):

--- a/tests/system/test_model_spectra.py
+++ b/tests/system/test_model_spectra.py
@@ -38,7 +38,7 @@ class TestModelSpectrumModel:
         for j, transition in enumerate(modelled_transitions):
             modelled_transition = [transition["name"], float(transition["E"])]
             assert modelled_transition in get_match_transitions, \
-                f"Missing transition for {transition["element"]}: {modelled_transition}"
+                f"Missing transition for {transition['element']}: {modelled_transition}"
 
     def test_linear_sigma_model(self):
         sigma_params = np.loadtxt("src/EVA/databases/detectors/energy_resolution_linear.txt",

--- a/tests/system/test_run_corrections.py
+++ b/tests/system/test_run_corrections.py
@@ -50,4 +50,4 @@ class TestRunCorrections:
                 corrected = raw[detector].x
 
             assert np.array_equal(corrected, run.data[detector].x), (f"energy correction failed when correcting "
-                                                                   f"{" and ".join(e_corr_which)}")
+                                                                   f"{' and '.join(e_corr_which)}")

